playwright.config.ts

```diff
@@ -13,8 +13,8 @@ export default defineConfig({
   testDir: './playwright-e2e',
   /* Run tests in files in parallel */
   fullyParallel: true,
-  timeout: 450000,
-  expect: { timeout: 450000 },
+  timeout: 10000,
```

Comment:
Getting loads of timeouts when I run these.

I restored the original 45000, and they all passed.  Perhaps keep the larger timeout?

Reply:
changed to 25000 to see if this is long enough